src/scales/scale.category.js

```diff
@@ -53,9 +53,6 @@ module.exports = function(Chart) {
 			var data = me.chart.data;
 			var isHorizontal = me.isHorizontal();
 
-			if (data.yLabels && !isHorizontal) {
```

Comment:
Is it really what we want? This test has been explicitly added in #3284 to fix #3278.

TBH, I'm not sure to understand why #3284 in the first place :)